### PR TITLE
Prepare for 1.1.0 release

### PR DESCRIPTION
**Pull request by tgraf**
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/4592)

### CODE REVIEW

**Comment by tgraf** [comment]
test-me-please

File: examples/kubernetes/1.10/cilium-crio-ds.yaml [hunk 1]
```diff
@@ -50,7 +50,7 @@ spec:
                 optional: true
                 key: clean-cilium-state
       containers:
-      - image: docker.io/cilium/cilium:v1.1.0-rc4
+      - image: docker.io/cilium/cilium:v1.1.0
         imagePullPolicy: Always
```
Comment (aanm) [inline examples/kubernetes/1.10/cilium-crio-ds.yaml]: `imagePullPolicy: IfNotPresent`, change it in the `.sed` file for the 1.1 branch and regenerate again

**Review by aanm — DISMISSED** [review]

File: examples/kubernetes/1.10/cilium-crio-ds.yaml (hunk 1 above)
Comment (tgraf) [reply examples/kubernetes/1.10/cilium-crio-ds.yaml]: Good catch

**Comment by tgraf** [comment]
test-me-please

**Comment by tgraf** [comment]
@aanm noticed missing quotes for proper rst rendering. We missed this for the rc releases as well

**Pull request merged**